mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Utils.kt

```diff
@@ -0,0 +1,34 @@
+package org.mifos.mobilewallet.mifospay.utils
+
+import android.app.Activity
+import android.content.Context
+import android.content.res.Resources
+import android.view.inputmethod.InputMethodManager
+import java.text.NumberFormat
+import java.util.*
+
+object Utils {
+
+    @JvmStatic
+    fun hideSoftKeyboard(activity: Activity) {
+        val view = activity.currentFocus
+        val imm =
```

Comment:
Just rename this variable imm to it's elaborate camel case form. Rest looks good.